### PR TITLE
Test sleef

### PR DESCRIPTION
**Pull request by xsacha**
* I have checked that this pull request contains only
  `.travis.yml`/`appveyor.yml` changes or commits merged from the `pkg.template` branch. All other changes send
  to https://github.com/ruslo/hunter. **[Yes]**
Other changes were sent to ruslo/hunter but had to be merged in for testing

* I have checked that no toolchains removed from CI configs, they are commented
  out instead so other developers can enable them back easily and to simplify
  merge conflict resolution. **[Yes]**

* I have checked that for every commented out toolchain there is a link to the
  broken CI build page or to the minimum compiler requirements documentation
  so other developers can figure out what was the problem exactly. **[Yes]**

### CODE REVIEW

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -394,6 +394,7 @@ hunter_default_version(roscpp_core VERSION 0.6.9-p0)
 hunter_default_version(rospack VERSION 2.5.0-p0)
 hunter_default_version(sds VERSION 2.0.0)
 hunter_default_version(sentencepiece VERSION 0.1.8-p1)
+hunter_default_version(sleef VERSION 3.3.1-p0)
```
Comment (ruslo) [inline cmake/configs/default.cmake:397]: >  All other changes send to https://github.com/ruslo/hunter.

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by ruslo** [comment]
>Other changes were sent to ruslo/hunter but had to be merged in for testing

To test https://github.com/ruslo/hunter/pull/1778 I need `pkg.sleef` to be ready and to make it ready I have to merge this pull request. And you're claiming that to merge this pull request we have to merge https://github.com/ruslo/hunter/pull/1778 :) Don't you see circular dependency here?

In doubts just follow the book blindly:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests-tests

**Comment by xsacha** [comment]
To add the package it wanted the test results done already

**Comment by ruslo** [comment]
Results from **your** branch `test.sleef`:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#run-remote-tests

**Comment by xsacha** [comment]
Sorry, this is actually much harder than it needs to be.
In about 10 minutes I can create the package and test it locally but creating all these extra repositories and test scripts, copying hash values to other repositories and seeing if it set up the correct way is too time consuming.

There needs to be a script that does all this boilerplate work (eg. Branch from template, sed script, run tests, etc).

If this wasn't my job, I would have given up straight away. But this is my job, I've been working with this for more than a year and I still have trouble so to me this means it is too hard.

My boss suggested just having the fork of Hunter with all the required packages added and not bother adding it to the GitHub, which I may have to consider.

You must also find this work tedious since you now require the submitter to do all this leg work. How it worked previously was very quick for me.

**Comment by ruslo** [comment]
> There needs to be a script that does all this boilerplate work (eg. Branch from template, sed script, run tests, etc).

Feel free to work on it, constructive suggestions are welcome as usual.

Just work your info:
- https://github.com/ruslo/hunter/issues/449
- https://github.com/ruslo/hunter/issues/138
- https://github.com/hunter-packages/hunter-bot
- https://github.com/hunter-packages/hunter-bot-elixir

> How it worked previously was very quick for me.

It was quite a long time ago and was pretty time consuming for me. Peoples keep submitting untested code and want me to fix everything.

**Pull request closed without merging**